packages/eslint-config-sentry-react/package.json

```diff
@@ -19,10 +19,10 @@
     "url": "https://github.com/getsentry/eslint-config-sentry/issues"
   },
   "peerDependencies": {
-    "eslint": ">=5",
-    "eslint-plugin-import": "~2.22.0",
-    "eslint-plugin-jest": "~22.17.0",
-    "eslint-plugin-react": "~7.15.1"
+    "eslint": "^7.11.0",
```

Comment:
I had problems with eslint7 fyi

Reply:
please what kind of problems did you have?

Reply:
yes we will have to wait a bit more